.pre-commit-config.yaml

```diff
@@ -40,3 +35,11 @@ repos:
             args:
                 - --target-version
                 - py37
+    - repo: https://github.com/PyCQA/flake8
+      rev: 4.0.1
+      hooks:
+      -   id: flake8
+          exclude: ^docs/.*
+          args: [
+              '--ignore=E121,E123,E126,E133,E203,E226,E231,E241,E242,E704,W503,W504,W505,W605'
```

Comment:
hmm, surprised we're doing this here and not in tox.ini or a flake8 config file - but that's future us problems :p